Xamarin.Forms.Platform.Android/Properties/AssemblyInfo.cs

```diff
@@ -23,7 +23,7 @@
 
 // These renderers are now registered via the RenderWithAttribute in the Android Forwarders project.
 // Note that AppCompat and FastRenderers are also registered conditionally in FormsAppCompatActivity.LoadApplication
-#if ROOT_RENDERERS
+#if ROOT_RENDERERS || TRUE
```

Comment:
This all-time be `true`. If it's done specifically, we need to remove that `#if` at all?